add instructions and screenshots for resize and updates in console. T… (#648)
EventStore/documentation: docs/cloud/ops/README.md

## Resizing cluster nodes

Clusters can be expanded on-demand, to accommodate database growth, through the [Cloud Console](https://console.eventstore.cloud/) and the [Event Store Cloud CLI](https://github.com/EventStore/esc).

> You can choose a larger or smaller node size, as well as a different cluster topology. See also the cloud [sizing guide](../provision/README.md#cloud-instance-sizing-guide) for general guidance.

Reviewer: We don't support changing cluster topology yet.

Author: I removed the bit about topology change

### Using the Cloud Console

To resize a cluster in the console, navigate to the clusters view and select _Resize Cluster_.

::: card
![cluster list](./images/Resize01DropDown.png)
:::

> On the detail page specify the new cluster size and click on _Resize Cluster_.

Reviewer 2: ```suggestion
On the detail page, specify the new cluster size and click on _Resize Cluster_.
```

::: card
![cluster_expand_detail](./images/Resize02Selection.png)
:::

You will get a prompt that this operation will require downtime. Click on _Start Operation_ to proceed.

::: card
![cluster_expand_detail](./images/Resize03DowntimeModalDialog.png)
:::

This will take you back to the cluster view where you will see that the resize is in progress.

::: card
![cluster_expand_detail](./images/Resize04Progress.png)
:::

> Once the resize operation has completed the new cluster size will show in the cluster view.

Reviewer 2: ```suggestion
Once the resize operation is complete, the new cluster size will show in the cluster view.
```

::: card
![cluster_expand_detail](./images/Resize05Complete.png)
:::

### Using the command line

> To resize a cluster with the command line, use the `clusters resize` command, where `--target_size` is the target instance size. For example, the size may be one of: F1, C4, M8, M16, M32, M64, M128.

Reviewer 2: are all fields in this example required? I would mention that. Is `id` self-explanatory? (I do not know, for instance, which id this would be if the project and org ids are separate fields). If these are the only possible values for target size, I would use "Possible values are F1, C4, M8, M16, M32, M64, M128."

> To resize a cluster with the command line, use the `clusters resize` command, where `--target_size` is the target instance size. For example, the size may be one of: F1, C4, M8, M16, M32, M64, M128.

Author: ```suggestion
To resize a cluster with the command line, use the `clusters resize` command, where `--target_size` is the target instance size. Possible values are: F1, C4, M8, M16, M32, M64, M128.
```

```bash
esc mesdb clusters resize \
    --target_size C4 \
    --id cn7dd2do0aekgb8nbf20 \
    --project-id cn62uolo0aegb5icm0bg \
    --org-id 9bsv0s4qu99g029v5560
```

## Upgrading EventStoreDB version

Clusters can be upgraded to a later version on-demand through the [Cloud Console](https://console.eventstore.cloud/) and the [Event Store Cloud CLI](https://github.com/EventStore/esc).

::: note
Limitations:

> * At this time clusters can only be updated to a minor version. Upgrading to a major version is part of a future release. Until that point, if you need to upgrade to a major servier version, you should perform a [backup](#manual-backup), and [restore](#restore-from-backup) to a new cluster. When restoring the backup, you can choose the EventStoreDB version you need. You can then switch your applications over to the connection string for the new cluster.

Reviewer 2: ```suggestion
* At this time, clusters can only be updated to a minor version of the current version. Upgrading to a major version is part of a future release. Until that point, if you need to upgrade to a major server version, you should perform a [backup](#manual-backup) and [restore](#restore-from-backup) to a new cluster. When restoring the backup, you can choose the EventStoreDB version you need. You can then switch your applications over to the connection string for the new cluster.
```

:::

### Using the Cloud Console

To upgrade a cluster in the console, navigate to the clusters view and select _Upgrade Cluster_.

::: card
![cluster list](./images/upgrade01ClusterDropDown.png)
:::

> On the detail page specify the new cluster version and click on _Upgrade Cluster_.

Reviewer 2: ```suggestion
On the detail page, specify the new cluster version and click on _Upgrade Cluster_.
```

::: card
![cluster_expand_detail](./images/upgrade02ClusterSelection.png)
:::

You will get a prompt that this operation will require downtime. Click on _Start Operation_ to proceed.

::: card
![cluster_expand_detail](./images/upgrade03DowntimeModalDialog.png)
:::

This will take you back to the cluster view where you will see that the upgrade is in progress.

::: card
![cluster_expand_detail](./images/upgrade04Progress.png)
:::

> Once the upgrade operation has completed the new cluster version will show in the cluster view.

Reviewer 2: ```suggestion
Once the upgrade operation is complete, the new cluster version will show in the cluster view.
```

::: card
![cluster_expand_detail](./images/upgrade05Complete.png)
:::

### Using the command line

> To upgrade a cluster with the command line, use the `clusters upgrade` command, where `--target_tag` is the version you want to upgrade to. This must include the full version e.g. 23.10.0.

Reviewer 2: Again, we should mention if the other fields are required.
